### PR TITLE
[IOTDB-2162] Simplify the recovery merge process

### PR DESCRIPTION
**Pull request by cigarl**
Simply the `MergeRecoveryTask`, we may just need to clean last info.

### CODE REVIEW

**Comment by wangchao316** [comment]
I  suggest we raise a jira firstly, and then raise pr.. So we can get a good tally of.

**Comment by cigarl** [comment]
> I suggest we raise a jira firstly, and then raise pr.. So we can get a good tally of.

Thanks for your reminding, I have created a [issue](https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-2162?filter=allopenissues).

File: server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/inplace/task/CleanLastMergeTask.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * RecoverCrossMergeTask is an extension of MergeTask, which resumes the last merge progress by
+ * scanning merge.log using LogAnalyzer and continue the unfinished merge.
+ */
+public class CleanLastMergeTask extends CrossSpaceMergeTask {
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/inplace/task/CleanLastMergeTask.java:35]: In master, we'd better rename merge to cross space compaction.
Another is inner space compaction.

Comment (cigarl) [reply server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/inplace/task/CleanLastMergeTask.java]: ok~ the related names have been changed.

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**